### DISCUSSION TITLE
zipfile.BadZipFile: File is not a zip file

### DISCUSSION THREAD

**Discussion by LYBAB2** [Trusted-AI/adversarial-robustness-toolbox#2243] [Q&A]
**Describe the bug**
When I was executing the notebook code in the example, I encountered a situation where the dataset could not be imported
**To Reproduce**
BadZipFile                                Traceback (most recent call last)
Cell In[26], line 1
----> 1 (x_raw, y_raw), (x_raw_test, y_raw_test), min_, max_ = load_mnist(raw=True)
      3 # Random Selection:
      4 n_train = np.shape(x_raw)[0]

File [d:\anaconda3\lib\site-packages\art\utils.py:1149](file:///D:/anaconda3/lib/site-packages/art/utils.py:1149), in load_mnist(raw)
   1137 """
   1138 Loads MNIST dataset from `config.ART_DATA_PATH` or downloads it if necessary.
   1139 
   1140 :param raw: `True` if no preprocessing should be applied to the data. Otherwise, data is normalized to 1.
   1141 :return: `(x_train, y_train), (x_test, y_test), min, max`.
   1142 """
   1143 path = get_file(
   1144     "mnist.npz",
   1145     path=config.ART_DATA_PATH,
   1146     url="https://s3.amazonaws.com/img-datasets/mnist.npz",
   1147 )
-> 1149 dict_mnist = np.load(path)
   1150 x_train = dict_mnist["x_train"]
   1151 y_train = dict_mnist["y_train"]

File [d:\anaconda3\lib\site-packages\numpy\lib\npyio.py:420](file:///D:/anaconda3/lib/site-packages/numpy/lib/npyio.py:420), in load(file, mmap_mode, allow_pickle, fix_imports, encoding, max_header_size)
    416 if magic.startswith(_ZIP_PREFIX) or magic.startswith(_ZIP_SUFFIX):
...
-> 1334     raise BadZipFile("File is not a zip file")
   1335 if self.debug > 1:
   1336     print(endrec)

BadZipFile: File is not a zip file
Output is truncated. View as a [scrollable element](command:cellOutput.enableScrolling?b4345af9-96f6-4593-8f25-f5b635621ffa) or open in a [text editor](command:workbench.action.openLargeOutput?b4345af9-96f6-4593-8f25-f5b635621ffa). Adjust cell output [settings](command:workbench.action.openSettings?%5B%22%40tag%3AnotebookOutputLayout%22%5D)...

**Comment by beat-buesser** [comment]
Hi @LYBAB2 Try deleting the file `mint.npz` in your local directory `.art` to start a new download. It looks like it got corrupted during the first download.

**Comment by beat-buesser** [comment]
I'm converting this issue into a Discussion.
